asynq/README.md

```
@@ -0,0 +1,71 @@
+Set `REDIS_ADDR` if Redis isn't on `localhost:6379`.
+
+## Trying it out
+
+```bash
+curl -X POST "http://localhost:3000/enqueue?email=jane@example.com&user=42"
+# {"enqueued":true,"task_id":"...","queue":"default"}
+```
+
+The worker terminal logs:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)